core/embed/extmod/modtrezorcrypto/modtrezorcrypto.c

```diff
@@ -69,21 +71,27 @@ STATIC const mp_rom_map_elem_t mp_module_trezorcrypto_globals_table[] = {
      MP_ROM_PTR(&mod_trezorcrypto_curve25519_module)},
     {MP_ROM_QSTR(MP_QSTR_ed25519),
      MP_ROM_PTR(&mod_trezorcrypto_ed25519_module)},
+#if !BITCOIN_ONLY
     {MP_ROM_QSTR(MP_QSTR_monero), MP_ROM_PTR(&mod_trezorcrypto_monero_module)},
+#endif
     {MP_ROM_QSTR(MP_QSTR_nist256p1),
      MP_ROM_PTR(&mod_trezorcrypto_nist256p1_module)},
     {MP_ROM_QSTR(MP_QSTR_groestl512),
```

Comment:
Why are we leaving this?

Reply:
I am leaving this, because it's quite hard to refactor the underlying trezor-crypto library. It does not have define macros for `USE_NISTP2561`, etc. We can do this later.